integration/runtime_test.go

```diff
@@ -157,9 +158,9 @@ func spawnGlobalDaemon() {
 			Scheme: testDaemonProto,
 			Host:   testDaemonAddr,
 		}
-		job := eng.Job("serveapi", listenURL.String())
-		job.SetenvBool("Logging", true)
-		if err := job.Run(); err != nil {
+
+		serverConfig := &apiserver.ServerConfig{Logging: true}
```

Comment:
Nah, integration calls `createRouter` directly anyway. I think it is probably easier just remove this field and use `true` and `false` values where it is needed for `createRouter`.